cli2gui/application/application.py

```diff
@@ -312,60 +392,160 @@ def createLayout(
 	Returns:
 		list[list[Element]]: list of widgets (layout list)
 	"""
-	argConstruct = []
+
+	args_layout = []
 	for section in buildSpec["widgets"]:
-		argConstruct = addItemsAndGroups(section, argConstruct, widgets)
+		args_layout = addItemsAndGroups(section, args_layout, widgets) # magic here
 
 	# Set the layout
-	layout: list[list[Element]] = [[]]
-	if isinstance(menu, list):
-		layout: list[list[Element]] = [[pySimpleGui.Menu([["Menu", menu]], tearoff=True)]]
-
-	layout.extend(
+	# NOTE(milahu) one big array : )
+	layout: list[list[Element]] = \
```

Comment:
Can you explain the reason for changing this please? 

Also I think older versions of python crash with var: type = value even with importing annotations